src/Surfnet/StepupSelfService/SelfServiceBundle/Service/SecondFactorService.php

```diff
@@ -111,6 +113,23 @@ public function revoke(RevokeCommand $command)
         return $result->isSuccessful();
     }
 
+    /**
+     * @param RemoteVetCommand $command
+     * @return bool
+     */
+    public function remoteVet(RemoteVetCommand $command)
```

Comment:
Simple as this method is, please consider covering it in test.

Reply:
Will have a look how valuable it will be to test glue code.